### PR TITLE
refactoring, added user list load error handling, redone Dashboard into stupid component

### PR DESCRIPTION
**Pull request by tatyana-budnyk**

### CODE REVIEW

**Review by LanaAvry — APPROVED** [review]
+

**Review by AnastasiiaHamaiunova — APPROVED** [review]
I have doubt about single quotes in the className

File: src/components/Avatar/Avatar.js [hunk 1]
```diff
@@ -4,16 +4,15 @@ import './Avatar.scss';
 
 const Avatar = (props) => {
   return (
-    <figure className="person-view">
-      <div className="person-avatar-and-name">
-        <div className="avatar-image"></div>
+    <figure className='person-view'>
+      <div className='person-avatar-and-name'>
+        <div className='avatar-image'></div>
```
Comment (Kaderimon) [inline src/components/Avatar/Avatar.js:9]: i think quotes should be changed

File: src/components/DashboardContent/NotificationsBell.js [hunk 2]
```diff
@@ -28,28 +28,26 @@ class NotificationsBell extends React.Component {
...
     return (
-    <div className="notification">
-        <div className="notification-bell">
-          <div className="notification-bell-self"
-            onClick={this.displayNotify}>
-            <i className="fa fa-bell-o"></i>
+    <div className='notification'>
+      <div className='notification-bell'>
+        <div className='notification-bell-self'
+          onClick={this.displayNotify}>
```
Comment (Kaderimon) [inline src/components/DashboardContent/NotificationsBell.js:34]: here as well

File: src/pages/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -1,27 +1,17 @@
...
-          <ListDevices />
-          <DashboardListUsers />
-        </div>
-      </div>
-    );
-  }
-}
+const Dashboard = () => (
+  <div className='dashboard-block'>
+    <div className='dashboard-devices-and-persons'>
```
Comment (Kaderimon) [inline src/pages/Dashboard/Dashboard.js:10]: and here

**Review by Kaderimon — DISMISSED** [review]

File: src/components/DashboardContent/NotificationsBell.js (hunk 2 above)
Comment (tatyana-budnyk) [reply src/components/DashboardContent/NotificationsBell.js:34]: this is done specifically on expert's request

**Pull request merged**